### DISCUSSION TITLE
Kuwait tv

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#93] [Show and tell]
this is the official stream of the kuwatian media services
https://www.media.gov.kw/LiveTV.aspx
we need the working stram link of this channels 
ktv 1:https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTV1
ktv 2: https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTV2
kuwait sport : https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTVSports
ktv drama : https://www.media.gov.kw/LiveTV.aspx?PanChannel=KhallikBilbait
ktv ithraa : https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTVEthraa
ktv al araby : https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTVArabe
ktv qurain : https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTVPlus
ktv sport plus : https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTVSportPlus
ktv al majlis : https://www.media.gov.kw/LiveTV.aspx?PanChannel=KTVSportExtra

**Comment by ztreamhd** [comment]
This is what I found: https://hiplayer.hibridcdn.net/p/kwmedia-kwtvsportsplus
If you copy and paste it on your browser, you'll download a M3U file with the media playlists (like this: https://kwmedia.hibridcdn.net/kwmedia/kwtvsportsplus_abr/kwmedia/kwtvsportsplus_1080p/chunks.m3u8) but unfortunately they won't play because you need a token, so the the link you need to go for would be something like this: https://hiplayer.hibridcdn.net/p/kwmedia-kwtvsportsplus?p=57&s=[UNIX Timestamp]&e=[UNIX Timestamp]&cf=[???]&h=[hash]

We should find a script that returns these tokens. PD: I used the KTV Sport+ link as an example.

**Reply by BellezaEmporium** [reply]
> This is what I found: https://hiplayer.hibridcdn.net/p/kwmedia-kwtvsportsplus If you copy and paste it on your browser, you'll download a M3U file with the media playlists (like this: https://kwmedia.hibridcdn.net/kwmedia/kwtvsportsplus_abr/kwmedia/kwtvsportsplus_1080p/chunks.m3u8) but unfortunately they won't play because you need a token, so the the link you need to go for would be something like this: [https://hiplayer.hibridcdn.net/p/kwmedia-kwtvsportsplus?p=57&s=[UNIX](https://hiplayer.hibridcdn.net/p/kwmedia-kwtvsportsplus?p=57&s=%5BUNIX) Timestamp]&e=[UNIX Timestamp]&cf=[???]&h=[hash]
> 
> We should find a script that returns these tokens. PD: I used the KTV Sport+ link as an example.

Just add .m3u8 to make the first link work in a player. Still won't do much due to tokens.

FYI, hiplayer plugin made it's appearance with Streamlink, and so my host supports it.

**Comment by BellezaEmporium** [comment]
Does https://kwmedia.hibridcdn.net/kwmedia/kwtv1_abr/playlist.m3u8 work ?

**Reply by ARAB-IPTV** [reply]
> Does https://kwmedia.hibridcdn.net/kwmedia/kwtv1_abr/playlist.m3u8 work ?

no
